pom.xml

```diff
@@ -261,8 +260,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.8.0</version>
                 <configuration>
-                    <source>${java.version}</source>
-                    <target>${java.version}</target>
+                    <source>1.8</source>
+                    <target>1.8</target>
```

Comment:
Why you don't want use ${java.version} variable?

Reply:
To make sure, we are really using java 8, independant of what is set in environment.